src/main/java/seedu/duke/Currency2D.java

```diff
@@ -0,0 +1,99 @@
+package seedu.duke;
+
+import seedu.duke.exception.FinanceException;
+
+
+import static seedu.duke.CurrencyList.currencyList;
+import static seedu.duke.newcurrency.PersonalCurrencyList.personalCurrencyList;
+
+public class Currency2D {
+    //can add up to 30 new currencies
+    static String[] currencies = {
+            "usd,us dollar,$,1.0",
```

Comment:
Instead of hard-coding each value, we could probably find a compilation somewhere. A better alternative is to replace this with an API call that is updated regularly.